.github/workflows/tests.yaml

```diff
@@ -53,5 +53,12 @@ jobs:
       - name: Get kratos container logs
         run: kubectl logs kratos-0 -c kratos -n testing
 
+      # Hack to overcome lack of tools (cat, tar) in the workload container
       - name: Get kratos config file
-        run: kubectl exec -it kratos-0 -c kratos -n testing -- cat /etc/config/kratos.yaml
+        run: |
+          juju ssh kratos/0 "PYTHONPATH=agents/unit-kratos-0/charm/venv/ python3 -c '
+          from ops import pebble
+          p = pebble.Client(\"/charm/containers/kratos/pebble.socket\")
+          f = p.pull(\"/etc/config/kratos.yaml\")
+          print(f.read())
+          '"
```

Comment:
Is looking for the config file an appropriate test for the rock? I'd do something that validates that the binary works. Anyway this is not a related to this PR since the integration tests were already doing this.

Reply:
I suppose it is there for debugging purposes, if something goes wrong then this would help debug it. Having said that, you should be able to get the same config file if you run the integration tests locally, so I am not opposed to removing this step.

Reply:
I suppose this part is missing:
```
        if: failure()
```